vaadin-messages-flow-parent/vaadin-messages-flow/src/main/java/com/vaadin/flow/component/messages/MessageListItem.java

```diff
@@ -342,6 +343,39 @@ private String getTheme() {
         }
     }
 
+    /**
+     * Adds one or more class names to this item. Multiple class names can be
+     * specified by using multiple parameters.
+     *
+     * @param classNames
+     *            the class name or class names to be added to the item
+     */
+    public void addClassNames(String... classNames) {
+        this.classNames.addAll(Arrays.asList(classNames));
+        propsChanged();
+    }
+
+    /**
+     * Removes one or more class names from this item. Multiple class names can
+     * be specified by using multiple parameters.
+     *
+     * @param classNames
+     *            the class name or class names to be removed from the item
+     */
+    public void removeClassNames(String... classNames) {
+        this.classNames.removeAll(Arrays.asList(classNames));
+        propsChanged();
+    }
+
+    /**
+     * Gets the set of CSS class names set on this item.
+     *
+     * @return a list of class names
+     */
+    public Set<String> getClassNames() {
```

Comment:
I guess this getter was added to have the classNames serialized and available client-side?

It would be possible to align with `themeNames` and add a [similar getter](https://github.com/vaadin/flow-components/blob/89514765ffae89fadaedf913e5378aafb9553e5e/vaadin-messages-flow-parent/vaadin-messages-flow/src/main/java/com/vaadin/flow/component/messages/MessageListItem.java#L337) that does the mapping here:
```java
// Used only for Jackson serialization
@JsonGetter
private String getClassName() {
    if (classNames.isEmpty()) {
        return null;
    } else {
        return classNames.stream().collect(Collectors.joining(" "));
    }
}
```

Then the connector changes wouldn't be necessary either

Reply:
Right, it does make sense to align this. I will update the PR.

Reply:
Updated. Also, added `hasClassName()` method to align the API.

Reply:
I guess the same should be done to the avatar PR?
https://github.com/vaadin/flow-components/pull/5562

Reply:
@DiegoCardoso Thanks, I will update it accordingly.